entrypoint.sh

```diff
@@ -35,7 +35,15 @@ function convertToKeyValuePairs() {
 }
 
 function getRunningWorkflowIds() {
-  jq ".workflow_runs | .[] | select(.head_branch==\"${branch}\" and .head_repository.full_name==\"${repo}\" and .status==\"in_progress\" or .status==\"queued\" or .status== \"waiting\") | .id " | grep -v "${GITHUB_RUN_ID}" || :
+  local workflow_ids=$(jq ".workflow_runs | .[] | select(.head_branch==\"${branch}\" and .head_repository.full_name==\"${repo}\" and .status==\"in_progress\" or .status==\"queued\" or .status== \"waiting\") | .id ")
+  local condition="<"
+  for id in $workflow_ids; do
+    if [[ "$id" -gt "$GITHUB_RUN_ID" ]]; then
```

Comment:
would it be easier to move this to https://github.com/rokroskar/workflow-run-cleanup-action/blob/4e83cfafee097e787449c764fb6030c5dded55f9/entrypoint.sh#L68 and avoid having two if statements?

Reply:
I can understand your idea. 
I have two reasons why I don't like it.
* logically the extraction of the id's should be done within `getRunningWorkflowIds`. An additional step to filter out newer workflows would add complexity somewhere where it logically doensn't belong.
* the extracted `workflow_ids` are typically not huge. When we assume that n=50 (which is very very big in my opinion.. why should anyone have 50 workflows running on the same branch) then the doubled execution time is still neglectable.

Personally I'd ignore the very minor extra computation in favor of clear structure and logical grouping. But it's your repo. If you want me to change it i'll do..

Reply:
sure that's fine with me if you prefer to keep it here